x-pack/plugin/inference/src/internalClusterTest/java/org/elasticsearch/xpack/inference/integration/TestInferenceServicePlugin.java

```diff
@@ -42,7 +42,7 @@ public class TestInferenceServicePlugin extends Plugin implements InferenceServi
 
     @Override
     public List<Factory> getInferenceServiceFactories() {
-        return List.of(TestInferenceService::new);
+        return List.of(TestInferenceService::new, TestInferenceServiceClusterService::new);
```

Comment:
This will now let us test both the `service.get().isInClusterService()` case and the `else`.